dark-mode.css

```diff
@@ -505,32 +575,63 @@ body.dark-mode .rcx-options .rcx-option--selected:hover {
 
 /***** Buttons *****/
 
-/* Regular button style */
-body.dark-mode .main-content .rcx-button:not(.rcx-button--ghost) { /* Default */
-	background-color: var(--color-dark) !important;
+/* Default style for buttons */
+
+body.dark-mode .main-content .rcx-button:not(.rcx-button--primary, .rcx-button--danger) {
+	background-color: var(--color-darkest);
 }
-body.dark-mode .main-content .rcx-button:is(.rcx-button--ghost, .rcx-button):hover { /* Hovered or selected */
-	background-color: var(--color-darkest) !important;
+body.dark-mode .main-content .rcx-button:is(.rcx-button--icon) {
+	border-color: transparent !important;
+}
+body.dark-mode .main-content .rcx-button:hover,
+body.dark-mode .main-content .rcx-button.rcx-button--icon-info { /* Replaces :focus rule */
+	background-color: var(--color-dark);
 }
 
-/* Square (icon) button style */
-body.dark-mode .main-content .rcx-button--square:not(.rcx-button--ghost), /* Default */
-body.dark-mode .main-content .rcx-button--square:is(.rcx-button--ghost, .rcx-button):hover {  /* Hovered or selected */
+/* Same style, for unusual buttons/icons/etc */
```

Comment:
Not sure how I feel about these comments yet. I understand their use though, so let's try them out for a bit and see what everyone thinks. My only concern is that they may become unwieldy.

Reply:
If it was only up to me, I would almost force them, to have a comment/explanation on each rule, that makes it easier to modify the file, but you're right, that can be heavy too !

Reply:
I definitely agree that explanation/comment would be good. The longer I've worked with the CSS, especially as RocketChat versions have changed, the more I want to think of a way to better organize the CSS code. 

Originally the idea was to group the rules by the location of the elements they styled, but so many various elements share classes that it doesn't completely solve the problem.

I'm very open to ideas though - what do you think is a good method of using comments to document the rules? 

Maybe something like:
```
/* main chat window - button - 5.0.0 */
body.dark-mode .main-content .rcx-button--square:not(.rcx-button--ghost), {
```

On that particular example, someone semi-familiar with Rocket.Chat's CSS can deduce what that rule does just by the class names. But it'll be more helpful on the rules with multiple generated class names.

Curious for your thoughts.

Reply:
By the way, I'm not thinking of the format - your tabular format is also fine - I'm asking more about what content you think would be best.

Reply:
I totally agree with your comment, and also with the fact that it became messy with RocketChat evolutions and our tries to organize things by sections

I like the idea, however it can quickly become heavy too if we put the location ("main chat window") in every comment 😅 I think I'd keep some "sections" to separate panels, windows, etc. but less than today
 Version number for each rule is a good idea to track things ! 

On that PR I tried to use CSS variables a lot, to be able to remove (or avoid adding) new rules that we would have to change/sort every time, that's also a way to go, maybe we can reduce our number of rules